CONTRIBUTING.md

```diff
@@ -62,16 +62,28 @@ Pull requests (PRs)
 - During a squash/fixup, use `exec make -C build unittest` between each
   pick/edit/reword.
 
-### Stages: WIP, RFC, RDY
+### Stages: Draft and Ready for review
 
-Pull requests have three stages: `[WIP]` (Work In Progress), `[RFC]` (Request
-For Comment) and `[RDY]` (Ready).
+Pull requests have two stages: draft and ready for review
 
-1. `[RFC]` is assumed by default, **do not** put "RFC" in the PR title (it adds
+1. ready for review is assumed by default, **do not** put "RFC" in the PR title (it adds
    noise to merge commit messages).
-2. Add `[WIP]` to the PR title if you are _not_ requesting feedback and the work
+2. Mark your PR as _draft_ if you are _not_ requesting feedback and the work
    is still in flux.
-3. Add `[RDY]` to the PR title if you are _done_ and only waiting on merge.
+
+### The `RFC` tag
```

Comment:
After a surprise team fight with @bfredl, I'd suggest using [RFD] (for "request for discussion") instead since that is close enough to RFC to be understandable while still being different from our previous (documented) meaning:

However, you can and should use the label `[RFD] ("Request For Discussion") for speculative, intrusive PRs that need broader discussion on its architecture or API.

Reply:
hum I haven't followed that discussion but RFC is an established acronym (thinking IETF etc) while it's the first time I see RFD. Not sure it's worth changing and I have trouble understanding the difference between our current RFC and the future RFD.

Reply:
Like RFC used to include the nitty gritty details while RFD is more about high-level directions of the PR ?

Reply:
Yes, RFC used to (and was argued still does) mean "ready for review", which should be the default state. (That other projects use it differently is unfortunate but does not change how Neovim uses it.) Suddenly changing that was argued to be needlessly disruptive -- hence we agreed on the compromise to use RFD for the need for more high-level discussion (for now; can always revisit after the dust has settled).

Reply:
Do we go for RFC then ?